app/config/index.js

```diff
@@ -40,7 +40,7 @@ const schema = Joi.object({
   googleTagManagerKey: Joi.string().allow(null, ''),
   isDev: Joi.boolean().default(false),
   port: Joi.number().default(3000),
-  serviceName: Joi.string().default('Annual health and welfare review of livestock'),
+  serviceName: Joi.string().required(),
```

Comment:
I think there are some defaults in here that we will need to ensure are still set in the main config object. This might help as I've been doing similar in the apply journey. https://github.com/DEFRA/ffc-ahwr-farmer-apply/pull/425

Reply:
I moved this one out to the config yeah - it seemed wrong that the schema was in control of the service name.

Reply:
In some cases the defaults can be removed, such as `isDev` which is set from `isDev: process.env.NODE_ENV === 'development'` so will always be a boolean anyway. But in others such as `cache.options.host` we seem to be relying upon the defaults being added?

Reply:
So the reason I had to make this change for serviceName is because we are now exporting the config object, rather than the result of running it through the schema.

This meant that the value just wasnt being set, so I moved it out of the schema and into the config (where it ought to have been in the first place).

This got the tests working again - I dont disagree there are heavy changes we need to make with the config, but I don't think this ticket is the right place to do them. This was just meant to be a minor quality of life change, alongside the real work which was changing the way we export functions and objects from the routes files.